core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -36,11 +34,12 @@ import org.apache.ivy.core.retrieve.RetrieveOptions
 import org.apache.ivy.core.settings.IvySettings
 import org.apache.ivy.plugins.matcher.GlobPatternMatcher
 import org.apache.ivy.plugins.resolver.{ChainResolver, IBiblioResolver}
-
 import org.apache.spark.SPARK_VERSION
 import org.apache.spark.deploy.rest._
 import org.apache.spark.util.{ChildFirstURLClassLoader, MutableURLClassLoader, Utils}
 
+import scala.collection.mutable.{ArrayBuffer, HashMap, Map}
```

Comment:
minor: please move this back up there and preserve the spacings as before (see style guide)